integrations/tensorflow/e2e/keras/applications/applications_benchmark.py

```diff
@@ -0,0 +1,120 @@
+# Lint as: python3
```

Comment:
Nit: I think the name "_benchmark.py" here is somewhat misleading because running this doesn't actually benchmark anything AFAICT. Maybe call these "_benchmark_artifacts.py" or something?

Reply:
Yeah, makes sense. Done.